administration/serializers.py

```diff
@@ -19,7 +20,7 @@ def get_phone_number(self, user):
         except ObjectDoesNotExist:
```

Comment:
I'm not sure you'll get `ObjectDoesNotExist` exception in case there are no `profile` instance for the user. Maybe it will be something about attribute etc.

If you haven't checked behavior here, please do this.

Reply:
Thank you for feedback. I've just fix bug with import in this file. But I'll tell the author of this function about your advice.